Simvex_FE/src/pages/cad/componenets/SceneObject.jsx

```
@@ -0,0 +1,174 @@
+    );
+
+    return () => {
+      // 클린업
+      if (model) {
+        model.traverse((child) => {
+          if (child.geometry) child.geometry.dispose();
+          if (child.material) {
+            if (Array.isArray(child.material)) {
+              child.material.forEach(mat => mat.dispose());
+            } else {
+              child.material.dispose();
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)